nimbus-eth1/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM debian:testing-slim AS build
+
+SHELL ["/bin/bash", "-c"]
+
+RUN apt update \
+ && apt -y install build-essential git-lfs
+
+
+RUN ldd --version ldd
+
+ADD . /root/nimbus-eth1
+
+RUN cd /root/nimbus-eth1 \
+ && make -j$(nproc) update \
+ && make -j$(nproc) V=1 LOG_LEVEL=TRACE nimbus
+
+# --------------------------------- #
+# Starting new image to reduce size #
+# --------------------------------- #
+FROM debian:testing-slim as deploy
+
+SHELL ["/bin/bash", "-c"]
+RUN apt update \
+ && apt -y install build-essential
+RUN apt update && apt -y upgrade
+
+RUN ldd --version ldd
+
+RUN rm -rf /home/user/nimbus-eth1/build/nimbus
+
+COPY --from=build /root/nimbus-eth1/build/nimbus /home/user/nimbus-eth2/build/nimbus
```

Comment:
wrong user path no?
it should be /home/user/nimbus-eth1/...

Reply:
Fixed.